src/uni_modules/wot-ui/components/wd-button/index.scss

```
@@ -132,30 +132,30 @@ $button-line-height-small: var(--wot-button-line-height-small, $typography-body-
-$button-left-right-padding-medium: var(--wot-button-left-right-padding-medium, $padding-extra-loose) !default;
-// large 尺寸左右内边距
-$button-left-right-padding-large: var(--wot-button-left-right-padding-large, $padding-super-loose) !default;
-// mini 尺寸上下内边距
-$button-up-down-padding-mini: var(--wot-button-up-down-padding-mini, $padding-super-tight) !default;
-// small 尺寸上下内边距
-$button-up-down-padding-small: var(--wot-button-up-down-padding-small, $padding-extra-tight) !default;
-// medium 尺寸上下内边距
-$button-up-down-padding-medium: var(--wot-button-up-down-padding-medium, $padding-tight) !default;
-// large 尺寸上下内边距
-$button-up-down-padding-large: var(--wot-button-up-down-padding-large, $padding-main) !default;
+// mini 尺寸内边距
```

Stylelint (17.7.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)